### PR TITLE
DAS-1442 - Add HOSS/MaskFill chained services.yml entry to production.

### PR DESCRIPTION
**Pull request by owenlittlejohns**
## Jira Issue ID

DAS-1442 (TRT-164)

## Description

This PR mimics the UAT entry in `services.yml` to add a production entry for the chained HOSS/MaskFill service.

## Local Test Steps

Not testable outside of the production environment.

## PR Acceptance Checklist
* [x] Acceptance criteria met
* [x] Tests added/updated (if needed) and passing (all tests passing)
* [x] ~Documentation updated (if needed)~

### CODE REVIEW

File: config/services.yml [hunk 1]
```diff
@@ -412,6 +412,36 @@ https://cmr.earthdata.nasa.gov:
       - image: !Env ${TRAJECTORY_SUBSETTER_IMAGE}
         operations: ['variableSubset', 'spatialSubset', 'shapefileSubset']
 
+  - name: sds/HOSS-maskfill
```
Comment (owenlittlejohns) [inline config/services.yml:415]: This entry is almost exactly copied from the UAT one with the following two differences:

* A different UMM-S reference.
* Not including a UAT test collection under the `collections` attribute for the service.

File: config/services.yml [hunk 2]
```diff
@@ -412,6 +412,36 @@ https://cmr.earthdata.nasa.gov:
...
+    data_operation_version: '0.16.0'
+    type:
+      <<: *default-turbo-config
+      params:
+        <<: *default-turbo-params
+        env:
+          <<: *default-turbo-env
+          STAGING_PATH: public/sds/HOSS
+    umm_s:
+      - S2300730272-XYZ_PROV
```
Comment (flamingbear) [inline config/services.yml:427]: verified same in prod as existing uat 
https://cmr.earthdata.nasa.gov/search/services.umm_json?concept_id=S2300730272-XYZ_PROV

**Review by flamingbear — APPROVED** [review]
LGTM.

**Review by chris-durbin — APPROVED** [review]

**Comment by chris-durbin** [comment]
Those vulnerabilities are on the dev dependencies, and it doesn't look there is an update available to lerna yet to fix them.

I verified running `better-npm-audit audit -p` says we're ok. Maybe we should change the test command to include that `-p` flag to only fail on production dependencies and not dev ones.

**Pull request merged**